service/models/product.py

```diff
@@ -11,18 +11,117 @@
 
 """
 
-class Product():
-  def create():
-    pass
+from flask import Flask
+from .model_utils import db,logger,Availability,DataValidationError
 
-  def read():
-    pass
+class Product(db.Model):
+
+  __tablename__= 'product'
+
+  app:Flask = None
+
+  id = db.Column(db.Integer,primary_key = True)
+  name = db.Column(db.String(64), nullable=False)
+  price = db.Column(db.DECIMAL,nullable=False)
+  status = db.Column(db.Enum(Availability),nullable=False,default=1)
+  pic_url = db.Column(db.Text,nullable=True)
+  short_desc = db.Column(db.Text,nullable=True)
+  
+  def create(self):
+    logger.info("Creating %s ...", self.name)
```

Comment:
Maybe it would be helpful to add `id`s in logging here? I'm not sure how much easier it would actually make our lives going forward, but I think it is helpful to immediately know how to address w//e item by it's primary key. Let me know.

Reply:
We don't get hold of id until the database actually saved the item to the table and returned the id.